internal/install/static_snapshot_yml.go

```diff
@@ -75,25 +75,41 @@ services:
       package-registry:
         condition: service_healthy
 
-  elastic-agent:
+  fleet-server:
     image: ${ELASTIC_AGENT_IMAGE_REF}
     depends_on:
       elasticsearch:
         condition: service_healthy
       kibana:
         condition: service_healthy
+    healthcheck:
+      test: "curl -f http://127.0.0.1:8220/api/status | grep HEALTHY 2>&1 >/dev/null"
+      retries: 12
+      interval: 5s
+    hostname: docker-fleet-server
+    environment:
+    - "FLEET_SERVER_ENABLE=1"
+    - "FLEET_SERVER_INSECURE_HTTP=1"
```

Comment:
I would rather see you run it without this flag. Why run it insecurely?

Reply:
For debugging purposes we can sniff network traffic and see requests/responses. It's not a production setup.